pytorch_lightning/trainer/trainer.py

```diff
@@ -250,7 +249,7 @@ def __init__(
             num_processes: number of processes for distributed training with distributed_backend="ddp_cpu"
 
             num_sanity_val_steps: Sanity check runs n validation batches before starting the training routine.
-                Set it to `-1` to run all batches in all validation dataloaders. Default: 2
```

Comment:
The defaults appear already in the docs signature

Reply:
yes, it is in signature and I would be careful about having it multiple times in docs especially since this is not checked against the signature so it can simply happen this side not would differ from the true one